components/brave_wallet/browser/fil_response_parser_unittest.cc

```diff
@@ -89,33 +89,21 @@ TEST(FilResponseParserUnitTest, ParseFilEstimateGas) {
               "Version": 0
           }
       })";
-  std::string gas_premium;
-  std::string gas_fee_cap;
-  int64_t gas_limit = 0;
-  EXPECT_TRUE(brave_wallet::ParseFilEstimateGas(ParseJson(json), &gas_premium,
-                                                &gas_fee_cap, &gas_limit));
-  EXPECT_EQ(gas_premium, "100466");
-  EXPECT_EQ(gas_fee_cap, "101520");
-  EXPECT_EQ(gas_limit, 2187060u);
 
-  gas_premium.clear();
-  gas_fee_cap.clear();
+  auto parse_result = brave_wallet::ParseFilEstimateGas(ParseJson(json));
+  EXPECT_TRUE(parse_result);
+  EXPECT_EQ(parse_result->gas_premium, "100466");
```

Comment:
As we are touching this it also makes sense to remove `brave_wallet::` prefix where it is possible. Everything in this file is in `brave_wallet` namespace.